doc/development-guide.md

```diff
@@ -93,11 +93,14 @@ In general, to get your changes approved by core members of OpenDevStack, please
 ensure the following:
 
 * Before you make a bigger change, open a ticket first and discuss what you want
-  to do before you actually do it.
-* Add the ticket to the [ODS project](https://github.com/orgs/opendevstack/projects), 
-  so we can track it.
-* Explain why this change is necessary / benefitial.
+  to do before you actually do it. E.g. explain why this change is necessary / benefitial.
```

Comment:
I was thinking as well, if it makes sense to create a developer mailinglist to send / discuss such design changes / enhancements or if a github issue and tracking it would be sufficient

Reply:
@rattermeyer For my taste, GitHub is sufficient. I'd like to have as few communication channels as possible ...